client/src/components/Header.tsx

```diff
@@ -4,48 +4,62 @@ import { useState } from "react";
 import styled from "styled-components";
 import { Colors } from "../colors/Colors";
 import { Shadows } from "../values/Shadows";
-import { Radiuses } from "../values/Radiuses";
-import { Spacings } from "../values/Spacings";
-import { Avatar, TextField } from "@mui/material";
+import { TextField } from "@mui/material";
 import ClearIcon from "@mui/icons-material/Clear";
 import SearchIcon from "@mui/icons-material/Search";
-import headerLogo from "/src/images/headerLogo.png";
+import headerLogo from '/src/images/logo-mark.svg'
+
+import AccountCircleRoundedIcon from '@mui/icons-material/AccountCircleRounded';
+import { Box } from '@mui/material';

```

Comment:
This can be imported with TextField in line 7

Reply:
TextField and Box are two separated things. TextField is like input in HTML and Box is like a div, so I need to use Box(div) in this case.